frontend/webpack.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   entry: './src/main.js',
   output: {
     filename: 'app.js',
-    path: path.resolve(__dirname, 'build')
+    path: path.resolve(__dirname, '../assets')
```

Comment:
I think we should not touch it. It can break anything. 
Just add yet another step into the pipeline.

Reply:
will do